### DISCUSSION TITLE
Running filters on data coming from Prometheus Scrape Metrics input

### DISCUSSION THREAD

**Discussion by d-ferenczi** [fluent/fluent-bit#6388] [Q&A]
Has anyone succeeded at running filters on data coming from Prometheus Scrape Metrics input?  
We can't even get the standard-output filter to display our data, whereas it works properly for other input plugins.  
Is Prometheus Scrape Metrics treated differently than other sources?

**Comment by agup006** [comment]
The Prometheus metrics are only compatible with metric-compatible output plugins: Prometheus Remote Write, Exporter, Splunk, InfluxDB, and OpenTelemetry (there may be more that I can't recall at the moment)

The stdout output plugin should work correctly, filter wise what type of operations are you looking to run?
